client/src/assets/terrains/terrain.ts

```diff
@@ -0,0 +1,16 @@
+import { Direction } from '../../shared/models/direction.enum';
```

Comment:
Codacy found an issue: ['Direction' is defined but never used.](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot/pullRequest?prid=5943029)